src/cli/bin.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict'
 
+require('pipe-args').load()
```

Comment:
@VictorBjelkholm yep, the committed tests are just as proof of concept. Once the feature is polished, I will add for all the tests for commands that must support stdin redirection. The link is super helpful, thanks!!